### PR TITLE
Fix: RTL issue with some of blocks.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

Trello - https://trello.com/c/DEeeGjSB/130-rtl-issue-with-some-of-blocks

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop each block and check is it working in rtl or not.
- Is there any block layout is broken or not. Such as Post Timeline, Content Timeline, Testimonial .etc

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by RajkiranBagal — APPROVED** [review]

File: output.rtl.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
+/* - Blockquote */
```
Comment (vrundakansara) [inline output.rtl.css:1]: Rename this file to `blocks.style.rtl.css`, and move it in the `/dist` folder.

File: output.rtl.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+.uagb-blockquote__skin-border blockquote.uagb-blockquote{
```
Comment (vrundakansara) [inline output.rtl.css:2]: Do we need a `.rtl` prefix here?

File: output.rtl.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  border-left-width:0px ;
+} 
+/* - Testimonial  */
+/* [dir="rtl"] .uagb-tm__arrow-outside.uagb-slick-carousel .slick-prev {
+  left: -45px ;
+  right: auto ;
+} */
+/* - Review star rating */
+.uagb_review_block .uagb_review_entry{
```
Comment (vrundakansara) [inline output.rtl.css:11]: Do we need a `.rtl` prefix here?

File: classes/class-uagb-helper.php [hunk 4]
```diff
@@ -271,11 +271,15 @@ public function print_stylesheet() {
...
-			<?php
-			ob_end_flush();
+				ob_start();
+				?>
+				<style id="uagb-style-frontend"><?php echo self::$stylesheet; //phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped ?></style>
+				<?php
+				ob_end_flush();
+
+			if ( is_rtl() ) {
+				wp_enqueue_style(  'uagb-style-rtl', UAGB_URL . 'output.rtl.css' );
```
Comment (vrundakansara) [inline classes/class-uagb-helper.php:281]: Add this in the file from where we add a static CSS file. This function is meant only for dynamic CSS.

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: classes/class-uagb-helper.php (hunk 4 above)
Comment (AbhijitNage123) [reply classes/class-uagb-helper.php]: Updated.

File: output.rtl.css (hunk 3 above)
Comment (AbhijitNage123) [reply output.rtl.css]: Updated

File: output.rtl.css (hunk 2 above)
Comment (AbhijitNage123) [reply output.rtl.css]: Updated

File: output.rtl.css (hunk 1 above)
Comment (AbhijitNage123) [reply output.rtl.css]: Updated

File: classes/class-uagb-block-js.php [hunk 5]
```diff
@@ -31,7 +31,7 @@ public static function get_testimonial_js( $attr, $id ) {
 
 			$dots   = ( 'dots' === $attr['arrowDots'] || 'arrowDots' === $attr['arrowDots'] ) ? true : false;
 			$arrows = ( 'arrows' === $attr['arrowDots'] || 'arrowDots' === $attr['arrowDots'] ) ? true : false;
-			$is_rtl = is_rtl();
+			$is_rtl       = is_rtl();
```
Comment (vrundakansara) [inline classes/class-uagb-block-js.php:34]: Get rid of this extra defined variable.

File: dist/blocks.style.rtl.css [hunk 6]
```diff
@@ -0,0 +1,79 @@
+/* - Blockquote */
```
Comment (vrundakansara) [inline dist/blocks.style.rtl.css:1]: Also, can we have a minified file here?

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: dist/blocks.style.rtl.css (hunk 6 above)
Comment (AbhijitNage123) [reply dist/blocks.style.rtl.css]: Yes, we can. Added

File: classes/class-uagb-block-js.php (hunk 5 above)
Comment (AbhijitNage123) [reply classes/class-uagb-block-js.php]: Updated

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**